File: java-spring/accounts-query-side-service/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/main/AccountsQuerySideServiceMain.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.main;

import net.chrisrichardson.eventstore.javaexamples.banking.web.AccountsQuerySideServiceConfiguration;
import org.springframework.boot.SpringApplication;

public class AccountsQuerySideServiceMain {

  public static void main(String[] args) {
    SpringApplication.run(AccountsQuerySideServiceConfiguration.class, args);
  }
}

File: scala-spring/accounts-query-side-service/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/main/AccountsQuerySideServiceMain.scala
package net.chrisrichardson.eventstore.examples.bank.web.main

import net.chrisrichardson.eventstore.examples.bank.web.AccountsQuerySideServiceConfiguration
import org.springframework.boot.SpringApplication

object AccountsQuerySideServiceMain  {

  def main(args: Array[String]) : Unit = SpringApplication.run(classOf[AccountsQuerySideServiceConfiguration], args :_ *)

}
